scripts/plotting/create_num_demos_plots.py

```diff
@@ -6,7 +6,7 @@
 import matplotlib.pyplot as plt
```

Comment:
I'm attempting to run some of these scripts with old results. Unfortunately it looks like we're going to have pickling issues if we try to do that:
```
Traceback (most recent call last):
  File "/Users/tom/phd/predicators/scripts/plotting/create_num_demos_plots.py", line 150, in <module>
    _main()
  File "/Users/tom/phd/predicators/scripts/plotting/create_num_demos_plots.py", line 112, in _main
    grouped_means, grouped_stds, _ = create_dataframes(COLUMN_NAMES_AND_KEYS,
  File "/Users/tom/phd/predicators/scripts/analyze_results_directory.py", line 159, in create_dataframes
    df = create_raw_dataframe(column_names_and_keys, derived_keys)
  File "/Users/tom/phd/predicators/scripts/analyze_results_directory.py", line 109, in create_raw_dataframe
    outdata = pkl.load(f)
  File "/Users/tom/.pyenv/versions/3.9.4/lib/python3.9/site-packages/dill/_dill.py", line 373, in load
    return Unpickler(file, ignore=ignore, **kwds).load()
  File "/Users/tom/.pyenv/versions/3.9.4/lib/python3.9/site-packages/dill/_dill.py", line 646, in load
    obj = StockUnpickler.load(self)
  File "/Users/tom/.pyenv/versions/3.9.4/lib/python3.9/site-packages/dill/_dill.py", line 636, in find_class
    return StockUnpickler.find_class(self, module, name)
ModuleNotFoundError: No module named 'predicators.src'
```

I think we can live with this, people will just have to check out an old version if they want to use old results... and everyone is more or less in between projects right now, so it's good that we're making this change now.

but, we should include a warning about this in the announcement that we'll make about this change.

Reply:
ahhh crap - i see. Well, making releases for all our major papers was a really good idea then!